modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py

```diff
@@ -470,7 +482,7 @@ class PandasOnRayDataframeColumnPartition(PandasOnRayDataframeVirtualPartition):
 
     Parameters
     ----------
-    list_of_blocks : list
+    list_of_partitions : list
```

Comment:
```suggestion
    list_of_partitions : Union[list, PandasOnRayDataframePartition]
```

To make consistent with parent-class. Possible, we also should change parameter description.

Reply:
The same as in https://github.com/modin-project/modin/pull/4695#discussion_r933324726

Reply:
Done.